app/_layouts/docs.html

```diff
@@ -80,10 +84,29 @@ <h5>{{nav.title}}</h5>
           <strong>Careful!</strong> You are browsing documentation for an outdated version of Kong. Go <a href="/docs/latest">here</a> to browse the documentation for the latest version.
         </div>
         {% endif %}
-
-      {{ content }}
+        
+        {{ content }}
 
       </div>
     </div>
   </div>
 </div>
+<style>
+  @media (max-width: 768px) {
+    .algolia-docsearch-suggestion--subcategory-column{
+      display: none !important;
+    }
+  }
+</style>
+<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"></script>
+<script type="text/javascript"> docsearch({
+    apiKey: '691286ef280379144076a4f8df47a1d1',
```

Comment:
Just double checking... This is the search-only API key, right?

Reply:
yes this is public facing search api key.